### DISCUSSION TITLE
I followed the HACS installation and it isn't working...

### DISCUSSION THREAD

**Discussion by Jace669** [ExperienceLovelace/ha-floorplan#15] [Q&A] [answered]
So... I am brand new to Home assistant … Sorry.

So I got it up and running through Hass.io on a raspberry Pi and I can see it through a private IP on my PC.

I went in and set a static IP, Installed SSH/Terminal, and a file editor.

installed HACS and basically got the first 4 steps done on this link:
https://experiencelovelace.github.io/ha-floorplan/docs/quick-start/

but now I am very lost and confused... I tried restarting the pi, and that did not help.  I tried adding cards(?) with code things...
![image](https://user-images.githubusercontent.com/51689050/104223795-1a297080-5412-11eb-8598-5f05c26c0b1b.png)

I'm just really dumb. A little guidance would be amazing.

**Comment by exetico** [comment, marked as answer]
Hi 👋

There's always a new beginning for everyone, when talking about tech-related things.

Could you try and follow [these steps](https://community.home-assistant.io/t/floorplan-now-available-as-a-lovelace-card/115489/323?u=exetico)? I know there's still a few things missing in the docs 🙏

Let me know how it goes 💪

**Reply by Jace669** [reply]
I will try it and get back to you in a few hours!

**Reply by Jace669** [reply]
Thank you exetico, I got the SVG to load! now onto the next steps to creating the SVG of my home & all the entities :O

**Comment by exetico** [comment]
Great! Fell free to close this 🎉

There's a guide on how to create SVG-files in the docs (or an old
reference, right now...). Good luck with that parts - that's much time
consuming 😂👏



//Tobias (from mobile)

tir. 12. jan. 2021 16.44 skrev Jace669 <notifications@github.com>:
